welcome/kubernetes-overview.adoc

```diff
@@ -0,0 +1,35 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="kubernetes-overview"]
+= Kubernetes overview
+include::_attributes/common-attributes.adoc[]
+:context: kubernetes-overview
+
+toc::[]
+
+Kubernetes is an open source container orchestration tool developed by Google. You can run and manage container-based workloads by using Kubernetes. The most common Kubernetes use case is to deploy an array of interconnected microservices, building an application in a cloud native way. You can create Kubernetes clusters that can span hosts across on-premise, public, private, or hybrid clouds.
```

Comment:
REVIEWERS: Though this looks like a new file, the content has been just moved from Arch and Getting Started books as part of the reorg effort. Any comments on this file will be addressed in subsequent PRs.